Scripts/build-archive.sh

```diff
@@ -27,6 +27,30 @@ if [ ! -d "$PRODUCTS_DIR/iOS/AppCenterDistributeResources.bundle" ] || \
   exit 1
 fi
 
+# Verify prefix.
+verify_prefix() {
+
+  # Get values.
+  frameworkPath=$1
+
+  # Get result from command.
+  result=$(nm -gU $frameworkPath | awk '{ print $3 }' | grep "_OBJC_CLASS_" | grep -v "_OBJC_CLASS_\$_MSAC" && return 1 || return 0)
```

Comment:
No need for sub-command here. Just put it into if without  `&& return 1 || return 0`